modules/persistent-storage-csi-sc-multiple-none.adoc

```diff
@@ -0,0 +1,50 @@
+// Module included in the following assemblies:
+//
+// persistent-storage-csi-sc-manage
+//
+:_content-type: PROCEDURE
+[id="persistent-storage-csi-sc-multiple-none_{context}"]
+= Absent or multiple default storage classes
+
+== Multiple default storage classes
+Multiple default storage classes can occur if you mark a non-default storage class as default and do not unset the existing default storage class, or you create a default storage class when a default storage class is already present. With multiple default storage classes present, any persistent volume claim (PVC) requesting the default storage class (`pvc.spec.storageClassName`=nil) gets the most recently default storage class, regardless of the default status of that storage class, and the administrator receives an alert in the alerts dashboard that there are multiple default storage classes, `MultipleDefaultStorageClasses`.
```

Comment:
maybe say  "gets the most recently **created** default storage class" instead of " gets the most recently default storage class" ?

Reply:
Good catch.